# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../exact-image/uint.patch
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- exact-image-1.0.2/bardecode/Tokenizer.hh.vanilla	2019-01-28 12:41:53.806175459 +0000
+++ exact-image-1.0.2/bardecode/Tokenizer.hh	2019-01-28 12:42:13.778175231 +0000
@@ -33,7 +33,7 @@
 {
 
     // Shall we include absolut position?
-    typedef std::pair<bool,uint> token_t; // (color,length in pixels)
+    typedef std::pair<bool, unsigned> token_t; // (color,length in pixels)
     typedef bool module_t;
     typedef double unit_t;
 
